Update NS entries to use nsone nameservers

anjannath opened the pull request.

cfergeau commented on crc.dev.zone:
```diff
@@ -2,15 +2,16 @@ $ORIGIN dev.
 
 crc     IN      SOA     ns1.osci.io. hostmaster.osci.io. (
-                        1739404800   ; serial
+                        1739404801   ; serial
```
current serial is 1740614400.
We won't be able to fix CI failures for both PRs and `main` though.

anjannath replied:
yeah, i have set it to 1740614401 now, `$current + 1` this should pass in the CI but will fail in main i think

cfergeau replied:
Yes, but hopefully this zone will go live soon, then we can re-run CI on `main` and this will be green.

cfergeau reviewed (commented):
serial issue, apart from this looks good to me!

anjannath committed:
update NS entries to use nsone nameservers

anjannath committed:
Set serial to 1740614401

the current serial is 1740614400 and since in
the last commit NS entries were updated it is
increased by 1
